Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -49,8 +49,8 @@ def __line__():
 ASSETS = "assets"
 EVENTS = "events"
 DATA_TYPES = [EVENTS, ASSETS]
-
-SECRET_REPLACEMENT_STRING = '<XX_REPLACED>'
+MASK = '<XX_REPLACED>'
+SEND_PREFIX = "send: b'"
```

Comment:
```suggestion
SEND_PREFIX = 'send: b"'
```
<img width="721" alt="image" src="https://github.com/demisto/content/assets/38749041/dd97ae25-8bbd-4fe6-8194-d8177dcd27f3">